<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="de" sourcelanguage="en">
<context>
    <name>generals</name>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="47"/>
        <source>Show errors</source>
        <translation>Fehler anzeigen</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="48"/>
        <source>Show warnings</source>
        <translation>Warnungen anzeigen</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="46"/>
        <source>Create layer</source>
        <translation>Layer erstellen</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="49"/>
        <source>Cancel</source>
        <translation>Abbrechen</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="52"/>
        <source>Show error log</source>
        <translation>Fehlerprotokoll anzeigen</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="53"/>
        <source>Upload xtf-log file</source>
        <translation>Pfad oder URL einer XTF-Error-Log Datei auswählen</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="54"/>
        <source>Show log for existing layer</source>
        <translation>Fehler-Log für vorhandenen Layer anzeigen</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="55"/>
        <source>Only layers created with this plugin can be selected</source>
        <translation>Nur mit diesem Plugin erstellte Layer können ausgewählt werden</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="91"/>
        <source>No valid file</source>
        <translation>Keine gültige Datei</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="85"/>
        <source>Could not get a valid XTF-Log file from specified Url</source>
        <translation>Keine gültige XTF-Log-Datei konnte von der angegebenen URL abgerufen werden</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="91"/>
        <source>No valid XTF-Log file at specified Path</source>
        <translation>Keine gültige XTF-Log-Datei unter angegebenem Pfad</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="51"/>
        <source>Error log</source>
        <translation>XTFLog-Checker Fehlerprotokoll</translation>
    </message>
</context>
</TS>
